src/themes/default/elements/icon.overrides

```diff
@@ -537,7 +582,7 @@ i.icon.weibo:before { content: "\f18a"; }
 i.icon.renren:before { content: "\f18b"; }
 i.icon.pagelines:before { content: "\f18c"; }
 i.icon.stack.exchange:before { content: "\f18d"; }
-i.icon.vimeo:before { content: "\f194"; }
+i.icon.vimeo.square:before { content: "\f194"; }
```

Comment:
With the addition of the new `vimeo` (the actual v) icon, makes more sense that the former icon now refers to `square vimeo`. That is my proposal.